spec/infinispan_ssl_spec.js

```diff
@@ -97,6 +133,16 @@ describe('Infinispan TLS/SSL client', function() {
     }
   }
 
+  function sslSniTrusted1() {
```

Comment:
Can we change this name, along with the `trusted1` prefix into something more meaningful? What are you testing exactly?